app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
list of skills should be an attribute of the concrete developer (not an attribute of the class)

Reply:
Firstly, you should create `SoftwareEngineer` class, 
which will be a parent for all other classes. 
Its constructor should take one parameter: `name` of the 
engineer. **Also, it should have `skills` attribute - 
a list of engineer's skills, such as `"Python"` or `"JavaScript"`.
`skills` should be initialised with empty list.**

Isn't it in the description or I missunderstand something?

Reply:
Maybe the description isn't occurate. But it's meant that an engineer (an instance of the class) should have a list of his/her skills

Reply:
We will fix the description to make it clearer